step3.cc

```diff
@@ -0,0 +1,24 @@
+class Solution {
+  public:
+      bool isValid(string s) {
+          const std::string open_brackets = "({[";
+          const std::string close_brackets = ")}]";
+          const std::unordered_map<char, char> close_to_open = {{')', '('}, {'}', '{'}, {']', '['}};
+  
+          std::stack<char> unclosed_brackets;
+          for (const char& c: s) {
```

Comment:
primitive は基本 copyの方が効率的です

```suggestion
          for (char c: s) {
```

Reply:
https://stackoverflow.com/questions/67541800/benefits-of-passing-const-reference-vs-values-in-function-in-c-for-primitive-t

こういう話ですね．
とりあえずconst+参照をしておけばいいと思っていたので，とても参考になりました．